python_benchmarks/global_params.py

```diff
@@ -25,7 +41,7 @@ def generate_input_sizes(dims: Union[int, List] = 2) -> List[Tuple]:
 
 
 # Datatypes to benchmark
-FLOAT_DTYPES = [torch.bfloat16, torch.float16, torch.float32]
+FLOAT_DTYPES = [torch.float16, torch.float32]
```

Comment:
Why `bfloat16` is dropped?

Reply:
It fails validation tests right now with the constant tolerances. So I dropped it for now, will add it back when I switch to `testValidate`.

Reply:
OK, that's fine but please leave a comment.